about.html

```diff
@@ -0,0 +1,173 @@
+                info@panafricanyouthconference.org</a
+            >
+          </p>
+        </div>
+      </div>
+
+      <section id="standOut">
+        <div class="wrapper">
+          <h2 class="section-heading">
+            Pan-African youth conference Logo
+          </h2>
+          <p>
+            The logo of Pan-African Youth conference was decided throu the logo competition from 8,june to 7 july.<br >
+            after online public voting and selection commitee's judging process, the logo from Nahom minas won the prize.<br />
+          </p>
+          <img loading="lazy"  src="https://www.panafricanyouthconference.org/wp-content/uploads/2022/01/PAYC-Logo-v2.1.png" alt="" title="PAYC-Logo v2.1" srcset="https://www.panafricanyouthconference.org/wp-content/uploads/2022/01/PAYC-Logo-v2.1.png 341w, https://www.panafricanyouthconference.org/wp-content/uploads/2022/01/PAYC-Logo-v2.1-300x145.png 300w" sizes="(max-width: 341px) 100vw, 341px" class="wp-image-243">
+        
+        </div>
+      </section>
+
+      <!-- OtherSfd - section -->
+      <section>
+        <div id="otherSfd" class="wrapper">
+          <h2 class="section-heading">
+            see the past Pan African youth conference
+          </h2>
+          <div class="other-programs">
+            <a
+              href="#"
+              class="program"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img src="images/PAYC1.jpg" alt="conference {
+                  
+              }" />
+              <span class="cover"></span>
+              <h3>2021</h3>
+              <p>conference</p>
+            </a>
+            <a
+              href="#"
+              class="program"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <img
+                src="images/Youth-Collaboration.jpg"
+                alt="SFD in Buenos Aires"
+              />
+              <span class="cover"></span>
+              <h3>2020</h3>
+              <p>conference</p>
+            </a>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <!-- Sponsors(partners)- section -->
+    <section id="sponsors">
+        <h2 class="section-heading">Partner</h2>
+        <div class="sponsors-list">
+          <a href="#">
+            <img src="images/MENA.png" title="Google" alt="Google"
+          /></a>
+          <a href="#">
+            <img src="images/glynn_logo.jpg" title="Nasa" alt="Nasa"
+          /></a>
+          <a href="#">
+            <img src="images/PASU_Logo.jpg" title="SpaceX" alt="SpaceX"
+          /></a>
+          <a href="#">
+            <img
+              src="images/Kellogg-logo.jpg"
+              title="Roscosmos"
+              alt="Roscosmos"
+          /></a>
+          <a href="#">
+            <img src="images/ASA_Logo-copy.png" title="airbnb" alt="airbnb" />
+          </a>
+        </div>
+      </section>
```

Comment:
Please remove the `partners` section for the `About` page. As it's not part of the design. It only should appear on the `Home` page.